### PR TITLE
Add landing message. #1.

### PR DESCRIPTION
**Pull request by mathGeek22**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,5 +8,6 @@
 </head>
   
 <body>
+    <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (markquark125) [inline index.html:11]: Remove 'Enjoy your stay.' and make 'Welcome to Potionfy!' red, please?

Comment (mathGeek22) [reply index.html:11]: koolio, can do.

**Review by markquark125 — CHANGES_REQUESTED** [review]
Please make the requested changes. Thanks!

**Review by markquark125 — APPROVED** [review]
Noice.

**Pull request merged**